db/schema.rb

```diff
@@ -139,6 +139,25 @@
     t.uuid "auto_established_claim_id", null: false
   end
 
+  create_table "disability_compensation_job_statuses", id: :serial, force: :cascade do |t|
+    t.integer "disability_compensation_submission_id", null: false
+    t.string "job_id", null: false
+    t.string "job_class", null: false
+    t.string "status", null: false
+    t.string "error_message"
+    t.datetime "updated_at", null: false
+    t.index ["disability_compensation_submission_id"], name: "index_disability_compensation_job_statuses_on_dsc_id"
+    t.index ["job_id"], name: "index_disability_compensation_job_statuses_on_job_id", unique: true
+  end
+
+  create_table "disability_compensation_submissions", id: :serial, force: :cascade do |t|
+    t.datetime "created_at", null: false
+    t.datetime "updated_at", null: false
+    t.integer "disability_compensation_id"
+    t.integer "va526ez_submit_transaction_id"
+    t.boolean "complete", default: false
+  end
+
```

Comment:
Revert these changes please.  This was unintentional, right?
I've noticed this `schema.rb` change happens when running `vets-api` via Docker.
I'm going to try to figure out why.

Reply:
Hey @jholton I will revert that, but yes it does seem to be to an unintended consequence of running `vets-api` via Docker